src/clue/clue.sass

```
@@ -62,6 +62,20 @@
               background-image: url("../assets/icons/delete/delete-hover.svg")
           &:active
               background-image: url("../assets/icons/delete/delete-active.svg")
+        .button-icon.share
+          &.visibility.public
+            background-image: url("../assets/icons/share/not-share.svg")
+            &:hover
+              background-image: url("../assets/icons/share/share-hover.svg")
+            &:active
+              background-image: url("../assets/icons/share/share-active.svg")
+        .button-icon.share
+          &.visibility.private
+            background-image: url("../assets/icons/share/share.svg")
```

```suggestion
            background-image: url("../assets/icons/share/share.svg")
```
Likewise, if the `visibility` is `private` we should show the `not-share` icon and then the `share` icon on hover/active.